### PR TITLE
Fix Pivot Reset -> core

### PR DESCRIPTION
**Pull request by YHerm**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/pivot/Pivot.java [hunk 1]
```diff
@@ -13,18 +13,25 @@ public class Pivot extends GBSubsystem {
...
 		this.pivotCommandsBuilder = new PivotCommandsBuilder(this);
-		this.resetFilter = new MedianFilter(PivotConstants.MEDIAN_FILTER_SIZE);
+		this.resetFilterRotations = new MedianFilter(PivotConstants.MEDIAN_FILTER_SIZE);
 
 		motor.resetPosition(PivotConstants.MINIMUM_ACHIEVABLE_ANGLE);
 		updateSignals();
+		resetRestFilter();
+	}
+
+	private void resetRestFilter() {
```
Comment (YHerm) [inline src/main/java/frc/robot/subsystems/pivot/Pivot.java]:
```suggestion
	private void resetResetFilter() {
```

File: src/main/java/frc/robot/subsystems/pivot/Pivot.java [hunk 2]
```diff
@@ -59,6 +66,10 @@ protected void setPower(double power) {
 		motor.setPower(power);
 	}
 
+	public void resetAngle(Rotation2d position){
+		motor.resetPosition(position);
+	}
```
Comment (YHerm) [inline src/main/java/frc/robot/subsystems/pivot/Pivot.java]: remove before merge

File: src/main/java/frc/robot/JoysticksBindings.java [hunk 3]
```diff
@@ -24,6 +27,9 @@ public static void configureBindings(Robot robot) {
 	private static void mainJoystickButtons(Robot robot) {
 		SmartJoystick usedJoystick = MAIN_JOYSTICK;
 		// bindings...
+		usedJoystick.B.onTrue(new InstantCommand(() -> robot.getPivot().resetAngle(PivotConstants.MINIMUM_ACHIEVABLE_ANGLE)));
+		usedJoystick.X.onTrue(robot.getPivot().getCommandsBuilder().moveToPosition(Rotation2d.fromDegrees(30), Rotation2d.fromDegrees(1)));
+		usedJoystick.Y.onTrue(robot.getPivot().getCommandsBuilder().stop());
```
Comment (YHerm) [inline src/main/java/frc/robot/JoysticksBindings.java]: remove

File: src/main/java/frc/robot/JoysticksBindings.java
Comment (YHerm) [inline src/main/java/frc/robot/JoysticksBindings.java:1]: format

**Review by bossyshrimp68 — DISMISSED** [review]
Fix yoav's requests

**Pull request merged**